src/test/resources/features/foi/FOIDispatch.feature

```diff
@@ -0,0 +1,18 @@
+@FOIDispatch @FOI @FOIWorkflow
+Feature: FOI Dispatch
+
+  Background:
+    Given I am logged into "CS" as user "FOI_USER"
+    When I create a "FOI" case and move it to the "Dispatch" stage
+    And I load and claim the current case
+
+  Scenario: User is able to complete the Dispatch stage
+    And I select "Yes" to do you want to dispatch the case
+    And I click the "Continue" button
+    Then the case should be moved to the "Soft Close" stage
+
+  @FOIRegression
+  Scenario: User is able to reject a case at the Dispatch stage
+    And I select "No" to do you want to dispatch the case
+    And I click the "Continue" button
+    Then the case should be moved to the "Approval" stage
```

Comment:
no rejection note for this scenario?

Reply:
I think I'll have to raise a bug, a rejection note isn't produced when the case is sent back